include/drm/drm_framebuffer.h

```diff
@@ -200,6 +203,10 @@ struct drm_framebuffer {
 	 * universal plane.
 	 */
 	int hot_y;
+	/**
+	 * @internal_flags: Framebuffer flags like DRM_FRAMEBUFFER_HAS_HANDLE_REF.
+	 */
+	unsigned int internal_flags;
```

Comment:
Move this so it comes right after `int flags;` instead, which more closely matches the intent of the upstream patch IMO since it's related fields.